cmd/snap-update-ns/utils.go

```diff
@@ -73,21 +73,17 @@ var (
 //
 // Directories are read only when they reside on file systems mounted in read
 // only mode or when the underlying file system itself is inherently read only.
-func IsReadOnly(dirFd int, dirName string) (bool, error) {
-	var fsData syscall.Statfs_t
-	if err := sysFstatfs(dirFd, &fsData); err != nil {
-		return false, fmt.Errorf("cannot fstatfs %q: %s", dirName, err)
-	}
+func IsReadOnly(dirFd int, dirName string, fsData *syscall.Statfs_t) bool {
```

Comment:
You don't need `dirFd` and `dirName` anymore.

Reply:
Ohh, that's right. Thank you, removed.